relay-config/src/config.rs

```diff
@@ -570,15 +569,19 @@ struct Http {
     encoding: HttpEncoding,
 }
 
+impl Http {
+    pub const DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD: f64 = 10.0;
```

Comment:
Let's make this a private const in this module, since this is exposed via the `Http::default()` implementation already.

Reply:
ok as you wish, putting it in Http made it more clear where it is used but also made using it a little bit less convenient.